package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "aes.js",
+  "name": "bsaes",
```

Comment:
`.js`?

Reply:
needed like this to link to https://www.npmjs.com/package/bsaes